core/src/main/scala/ai/lum/odinson/IdGetter.scala

```diff
@@ -7,6 +7,11 @@ trait IdGetter {
   def getSentId: String
 }
 
+class KnownIdGetter(docId: String, sentId: String) extends IdGetter {
```

Comment:
Can you please add a brief comment here?

Reply:
addressed, thanks!